centreon/www/widgets/src/centreon-widget-topbottom/src/MetricTop.tsx

```diff
@@ -60,19 +65,39 @@ const MetricTop = ({
   return (
     <>
       <Typography className={classes.resourceLabel}>
-        <strong>
-          #{inc(index)} {`${metricTop.parentName}_${metricTop.name}`}
-        </strong>
+        <Link
+          data-testid={`link to ${metricTop?.name}`}
+          style={{ all: 'unset' }}

```

Comment:
Use the `underline="hover"` prop in order to display a line while hovering the text then remove the `style={{ all: 'unset' }}`

Reply:
Are you suggesting the use of Link from Material-UI instead of the Link provided by React Router?